src/test/ratings.spec.ts

```diff
@@ -0,0 +1,184 @@
+import { ApolloServer } from 'apollo-server'
+import { expect } from 'chai'
+import { server } from '../index'
+import { describe } from 'mocha'
+import { GraphQLError } from 'graphql/error/GraphQLError'
+
+
+    
+after (()=> {
+    server.stop()
+})
+export const fetchRatings = `
+query Query {
+  fetchRatings {
+    user
+    sprint
+    quantity
+    quality
+  }
+}
+`
+
+export const LoginUser = `
+  mutation LoginUser($loginInput: LoginInput) {
+  loginUser(loginInput: $loginInput) {
+    token
+    user {
+      id
+      role
+    }
+  }
+}
+`
+
+export const AddRatings = `
+mutation Mutation($user: String!, $sprint: Int!, $quantity: String!, $quality: String!, $professionalSkills: String!, $remarkQuantity: String, $remarkQuality: String, $remarkProfessionalSkills: String) {
+  addRatings(user: $user, sprint: $sprint, quantity: $quantity, quality: $quality, professional_Skills: $professionalSkills, remark_quantity: $remarkQuantity, remark_quality: $remarkQuality, remark_professional_Skills: $remarkProfessionalSkills) {
+    user
+    sprint
+  }
+}
+`
+
+const updateRating = `
+mutation Mutation($user: String!, $sprint: Int) {
+  updateRating(user: $user, sprint: $sprint) {
+    user
+    sprint
+    quantity
+    remark_quantity
+  }
+}`
+
+ 
+
+describe('Manage Ratings', ()=>{
+    let testServer : ApolloServer
+    let admin: string
+    let coordinatorId: string
+    let coordinatorToken: string
+    let coordinatorRole: string
+    before (
+        async()=> 
+            setTimeout(async()=>
+            {
+                testServer = server
+
+                // login coordinator
+                const { errors: coordinatorError, data: coordinatorData } =
+            await testServer?.executeOperation({
```

Comment:
This function will always be available, there is no need to use the `?` to check